### PR TITLE
chore: bump devcontainer to node 18

### PR DESCRIPTION
**Pull request by jqshuv**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [x] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

I decited to bump this version because I've tried to open this repo in a Remote-Container in vscode and got the error that you need at least NodeJS 16.

## Test Plan

I've rebuilded the devcontainer and it worked without any errors.

image

### Test links

Deploy preview: https://deploy-preview-7726--docusaurus-2.netlify.app/

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
   "waitFor": "postCreateCommand", // otherwise automated jest tests fail
   "features": {
     "node": {
-      "version": "14"
+      "version": "16"
```
Comment (Josh-Cena) [inline .devcontainer/devcontainer.json:29]: Can we go directly to Node 18?

Comment (jqshuv) [reply .devcontainer/devcontainer.json:29]: I'll try it this evening but I think it should work if Docusaurus works on NodeJS 18.

Comment (jqshuv) [reply .devcontainer/devcontainer.json]: Yes it works and I already pushed it!
Screenshot_20220706-102832_Termius

**Review by Josh-Cena — APPROVED** [review]
Great, thanks!

I don't use devcontainers myself so it's always maintained by community contributors. Thanks for taking this up!

**Pull request merged**